modules/bibauthorid/lib/bibauthorid_dbinterface.py

```diff
@@ -2305,7 +2321,6 @@ def get_orcid_id_of_author(pid):  # get_orcids_by_pids
     '''
     return _select_from_aidpersoniddata_where(select=['data'], pid=pid, tag='extid:ORCID')
```

Comment:
This function was changed by you, not sure why the changes are not involved in this PR (maybe we changed it with Sam while fixing.

The type of the result was changed.

Please check `hoover` and the rest of the code for calls.